projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -202,6 +203,46 @@ export function asmCustomerLists(): void {
     });
 }
 
+export function asmB2bCustomerLists(): void {
+  const customerListsRequestAlias = asm.listenForCustomerListsRequest();
+  const customerSearchRequestAlias = asm.listenForCustomerSearchRequest();
+
+  cy.log('--> Starting customer list');
+  asm.asmOpenCustomerList();
+
+  cy.wait(customerListsRequestAlias)
+    .its('response.statusCode')
+    .should('eq', 200);
+
+  cy.wait(customerSearchRequestAlias)
+    .its('response.statusCode')
+    .should('eq', 200);
+
+  cy.get('cx-customer-list table').should('exist');
+  cy.get('cx-customer-list table').should('not.contain', 'Account');
```

Comment:
default selecting a customer list which is not b2b customer list. This list should not include account column.